### PR TITLE
Update Set-UnifiedGroup.md

### PR DESCRIPTION
**Pull request by Ashok-Lobo**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: exchange/exchange-ps/exchange/Set-UnifiedGroup.md [hunk 1]
```diff
@@ -843,6 +843,7 @@ The HiddenFromExchangeClientsEnabled switch specifies whether the Microsoft 365
 
 - To enable this setting, you don't need to specify a value with this switch. The Microsoft 365 Group is hidden from Outlook experiences. The group isn't visible in the Outlook left-hand navigation and isn't be visible in the global address list (GAL). The group name won't resolve during the creation a new message in Outlook. The group can still receive messages, but users can't search for or browse to the group in Outlook or Outlook on the web. Users also can't find the group by using the Discover option in Outlook on the web. Additionally, the HiddenFromAddressListsEnabled property will also be set to true to prevent the group from showing in the GAL and in the Offline Address Book (OAB).
 - To disable this setting, use this exact syntax: `-HiddenFromExchangeClientsEnabled:$false`. The Microsoft 365 Group is not hidden from Outlook experiences. The group will be visible in the GAL and other address lists. This is the default value.
+- If Microsoft 365 Groups are hidden from exchange clients, users cannot view the option to subscribe or unsubscribe to a Microsoft 365 Group.
```
Comment (JohanFreelancer9) [inline exchange/exchange-ps/exchange/Set-UnifiedGroup.md:846]:
```suggestion
- If Microsoft 365 Groups are hidden from Exchange clients, users cannot view the option to subscribe or unsubscribe to a Microsoft 365 Group.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Ashok-Lobo, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by Ashok-Lobo** [comment]
@JohanFreelancer9, committed the suggested change. thank you

**Comment by Ashok-Lobo** [comment]
@JohanFreelancer9 @chrisda @yogkumgit 

Please merge this PR. Thank you.

**Pull request merged**